lib/binance/futures/order.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Binance.Futures.Order do
```

Comment:
Wondering why we have this module `Binance.Futures.Order`, why not use this `Binance.Order` ?

Reply:
Because Spot and Futures returns different response @kimquy